src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return color + " " + number;
```

Comment:
```suggestion
        return "Ball with color " + color + ", number " + number;
```
make informative toString